api/turing/imagebuilder/imagebuilder_test.go

```diff
@@ -305,10 +357,9 @@ func TestBuildPyFuncEnsemblerJobImage(t *testing.T) {
 				tt.modelName,
 				tt.modelID,
 				tt.versionID,
-				tt.artifactURI,
+				testArtifactURI,
```

Comment:
Lol thanks for realising that `tt.artifactURI` is the same for all test cases and for replacing it with a single re-usable variable instead.